sdk/core/azure-core/src/main/java/com/azure/android/core/http/interceptors/ResponseValidationInterceptor.java

```diff
@@ -0,0 +1,45 @@
+package com.azure.android.core.http.interceptors;
```

Comment:
This interceptor is unique to Storage only, so we should put it in that package.

Reply:
Should we move it in the future or right now? I'm not sure if I recall correctly but I think we said we'd have these in core for now and move them once we started working on storage.

Reply:
I don't think I was there for that discussion. I would probably move them to the storage package now (creating any missing project structure).

Reply:
It was me suggested putting them in core and moving out eventually when we start do storage. Agree, lets create storage-package, which is needed anyway if we want to have these to be package internal.